Merge master staging bitrefill

Beerosagos opened the pull request:
Commit https://github.com/BitBoxSwiss/bitbox-wallet-app/commit/bc98b61705c21e7a4701425fe355cd22eb19435c changed the signature of `GetUnusedReceiveAddresses()`.
This was not automatically caught and fixed in the merge, but is fixed with an additional commit.

thisconnect reviewed (approved):
tested qt macos LGTM

Beerosagos commented:
@thisconnect I just realized the PR base was wrongly set to master instead of staging-bitrefill :see_no_evil: fixed now. 
Also fixed the nit.

PTAL :pray:
